app/models/item.rb

```diff
@@ -0,0 +1,7 @@
+class Item < ActiveRecord::Base
+  belongs_to :item_type
+  validates :name, :item_type_id, presence: true
+  validates :item_type_id, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
Are you able to `validate :item_type` as opposed to having to double check it, presence and numericality?